### DISCUSSION TITLE
Upgrade from 2.3.120

### DISCUSSION THREAD

**Discussion by secengsupport** [Security-Onion-Solutions/securityonion#9607] [Unsupported Versions]
I have recently inherited a distributed SO environment with a manager, six search nodes, and ten or so sensors scattered throughout the network. It is on 2.3.120 and I was looking to bring it up to something more current. I know how to do the upgrade but I was just looking for any "gotchas" I should be aware of going from x.x120 to x.x.190. I have read all of the update threads here and the release notes, checked the Elastic versions, etc. Anything major to watch out for? I was thinking that x.x.120 was a pretty good place to be starting from.

**Comment by cm-ops** [comment]
If the grid was installed prior to `2.3.60`, check Elastic auth to make sure it is `true.`. Otherwise this section of the soup docs will give a good idea  of items to look at  https://docs.securityonion.net/en/latest/soup.html?p#elastic-8

**Reply by secengsupport** [reply]
Thanks for the response. I did not get a notification that someone had replied so I just now saw it. 

It looks like we are good on Elastic auth. I think this was a rebuild from an old 1.x to 2.3.x, so was relatively new-ish. I guess my next question is that it all runs on Ubuntu 18.04. I know that has to be deprecated soon, especially with 2.4 coming. What does an OS upgrade look like in regards to SO impact? I know we ran into issues deploying some 20.04 sensors to an 18.04 management node (known issue) so backed them all down to 18.04 to match. I figure we might need to deal with the 18.04 issue as well.
